commands/Fun/diceroll.js

```diff
@@ -0,0 +1,17 @@
+const { Command } = require('klasa');
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, {
+			aliases: ['dr'],
+			description: 'Gives you a random dice roll from 1-10.',
+		});
+	}
+
+	run(message) {
+        const numbers = ['1', '2', '3', '4', '5', '6', '7', '8', '9', '10'];
```

Comment:
Should this not be maxed at 6?

Reply:
Not all dices are cubes, but if you think max of 6 is better, sounds good to me too